compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/KnownTargets.cpp

```diff
@@ -99,6 +103,19 @@ TargetAttr createTargetAttr(const TargetDetails &details, StringRef arch,
   for (int i = 0; i < wgp->mmaCount; ++i)
     mmaAttrs.push_back(MMAAttr::get(context, wgp->mmaOps[i]));
 
+  SmallVector<ScaledMMAAttr, 8> scaledMmaAttrs;
+  scaledMmaAttrs.reserve(wgp->scaledMmaCount);
+  for (int i = 0; i < wgp->scaledMmaCount; ++i) {
+    SmallVector<Type> inTy = ScaledMMAAttr::getSupportedInputTypes(context);
+    for (int j = 0; j < inTy.size(); j++) {
+      SmallVector<Type> outTy = ScaledMMAAttr::getSupportedOutputTypes(context);
+      for (int k = 0; k < outTy.size(); k++) {
```

Comment:
`for (Type outTy : outTys)` and similarly - I don't think you need `i`, `j`, and `k`